block-storage-attacher/tests/e2e/e2e-tests/utilscript.sh

```diff
@@ -23,7 +23,17 @@ export LANG=C.UTF-8
 # Load common functions
 . $SCRIPTS_FOLDER_PATH/common.sh
 
+function cleanupNode ()
+{
+   NodeIP = $1
+   commandstring = $2
+   JOB_NAME=$(LC_CTYPE=C cat /dev/urandom | base64 | tr -dc a-z0-9 | fold -w 32 | head -n 1)
+   $E2E_PATH/runon_worker.sh  $NodeIP  \"$commandstring\" >>$E2E_PATH/command_output 2>&1
```

Comment:
I think you need to pass the JOB_NAME to runon_worker.sh script as its referred there.